src/client/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -249,7 +249,7 @@ private async Task<IEnumerable<IAccount>> GetAccountsInternalAsync(ApiIds apiId,
             ICacheSessionManager cacheSessionManager, 
             CancellationToken cancellationToken)
         {
-            if (AppConfig.IsBrokerEnabled)
+            if (AppConfig.IsBrokerEnabled && ServiceBundle.PlatformProxy.CanBrokerSupportSilentAuth())
             {
                 var broker = ServiceBundle.PlatformProxy.CreateBroker(ServiceBundle.Config, null);
                 if (broker.IsBrokerInstalledAndInvokable())
```

Comment:
Android proxy already calls this method so now it will be called twice. Should probably remove it from the proxy and add it to the places where broker is used silently, or vice versa

Reply:
It is good to have optimization. But IMO, we should not do it in the hotfix. Hotfix should be simple reversal/just necessary changes.

Reply:
Understood, but the IsBrokerInstalledAndInvokable was also moved in the previous PR and it is a relatively simple thing to resolve. Otherwise we will need to make a new issue for this minor optimization.

Reply:
The method id removed from Android Proxy. Kept here as this base class is called by several platforms.

Reply:
Yep. Line 255 is needed for WAM broker too.
With current changes it makes sense. This code block will only be executed for non-iOS and if broker is enabled. And then we check if broker is actually installed.